iron-media-query.html

```diff
@@ -58,25 +58,26 @@
         type: Boolean,
         value: false
       },
-      
+
       /**
        * @type {function(MediaQueryList)}
-       */ 
+       */
       _boundMQHandler: {
         value: function() {
           return this.queryHandler.bind(this);
         }
       },
-      
+
       /**
        * @type {MediaQueryList}
-       */ 
+       */
       _mq: {
         value: null
       }
     },
 
     attached: function() {
+      this.style.display = 'none';
```

Comment:
God to know. Thanks.

On Tue, Nov 24, 2015, 6:08 PM Monica Dinculescu <notifications@github.com>
wrote:

> In iron-media-query.html
> <https://github.com/PolymerElements/iron-media-query/pull/27#discussion_r45822353>
> :
>
> >        _mq: {
> >          value: null
> >        }
> >      },
> >
> >      attached: function() {
> > +      this.style.display = 'none';
>
> And by slower I mean more expensive, but mobile github doesn't let me edit
> messages
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PolymerElements/iron-media-query/pull/27/files#r45822353>
> .
>